tests/resources/testcases/test_prinz_bias/testcase.yaml

```
@@ -0,0 +1,30 @@
+scenario: 'scenario.yaml'
+
+
+
+
+# reference sequence
+reference:
+  path: 'ref.fa'
+
+mode: Generic
+
+version: '4'
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)